### PR TITLE
added GSoD students to mentorship page

### PR DESCRIPTION
**Pull request by ritik-malik**
Hi @klumb @vchrombie ,
this PR refers to the issue #582

Fixes done till now :
* Changed the description to include *Google Summer of  Docs*
* Added new column named *Program* in the table & filled it accordingly
* Added data for GSoD students

For now I've put a place holder for *Project Description* since I'm not sure what to add.
Can you please review the above changes & guide me to *Project Description* data...

Signed-off-by: ritik-malik <ritik18406@iiitd.ac.in>

### CODE REVIEW

File: Community/alumni.md [hunk 1]
```diff
@@ -2,33 +2,61 @@
 
 # CHAOSS Mentorship Alumni
 
-The CHAOSS Community thanks all mentees and mentors who participated in Google Summer of Code and Outreachy.
+The CHAOSS Community thanks all mentees and mentors who participated in Google Summer of Code, Google Summer of Docs and Outreachy.
```
Comment (vchrombie) [inline Community/alumni.md:5]: Google Season of Docs

File: Community/alumni.md [hunk 2]
```diff
@@ -2,33 +2,61 @@
...
+
+Create a CHAOSS Community-wide Handbook
+
+
+#### Project Description
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam faucibus sapien nisl, sit amet semper urna imperdiet sit amet. Mauris tempor, dui eu placerat dapibus, sapien mi maximus purus, in porta nibh ante et turpis. Donec ac vestibulum enim. Donec ut iaculis lacus. Phasellus consectetur elementum vulputate. Donec pulvinar lectus sit amet condimentum sollicitudin. Vestibulum sit amet interdum tellus. Quisque vitae varius nibh, id lobortis quam.
+
+
+### Xiaoya Esther
```
Comment (vchrombie) [inline Community/alumni.md:48]: Xiaoya Xia

File: Community/alumni.md [hunk 3]
```diff
@@ -2,33 +2,61 @@
...
+
+Xiaoya Esther selected for GSoD 2020
```
Comment (vchrombie) [inline Community/alumni.md:50]: same as above comments

File: Community/alumni.md [hunk 4]
```diff
@@ -2,33 +2,61 @@
...
-|[Parth Sharma](#parth-sharma)|2019|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins|
-|[Bingwen Ma](#bingwen-ma)|2019|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins, Matt Germonprez|
-|[Aniruddha Karajgi](#aniruddha-karajgi)|2019|Implementing CHAOSS Metrics with Perceval|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
-|[Nishchith K Shetty](#nishchith-k-shetty)|2019|Support of Source Code Related Metrics|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
-|[Keanu Nichols](#keanu-nichols)|2018|Reporting of CHAOSS Metrics|Sean Goggins, Jesus Gonzalez-Barahona|
-|[Pranjal Aswani](#pranjal-aswani)|2018|Reporting of CHAOSS Metrics: Refactoring the existing code and extending the capabilities of the Manuscripts Project|Valerio Cosentino, Jesus Gonzalez-Barahona|
+|Student|Year|Program|Project|Mentors|
+|-------|----|-------|-------|-------|
+|[Jaskirat Singh](#Jaskirat-Singh)|2020|GSoD|Create a CHAOSS Community-wide Handbook|Georg J.P.link, Armstrong Foundjem, Matt Germonprez
```
Comment (vchrombie) [inline Community/alumni.md:12]: I think creating a new column for program doesn't give us any special benefits. I was thinking if program and year can be added as one column to maintain simplicity such as `GSoC 2018`, `Outreachy 2020`, `GSoD 2020`. I see most people use the same titles on their profiles too.

WDYT @klumb?

File: Community/alumni.md [hunk 5]
```diff
@@ -2,33 +2,61 @@
...
+Jaskirat Singh selected for GSoD 2020
+
+#### Project Title
+
+Create a CHAOSS Community-wide Handbook
+
+
+#### Project Description
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam faucibus sapien nisl, sit amet semper urna imperdiet sit amet. Mauris tempor, dui eu placerat dapibus, sapien mi maximus purus, in porta nibh ante et turpis. Donec ac vestibulum enim. Donec ut iaculis lacus. Phasellus consectetur elementum vulputate. Donec pulvinar lectus sit amet condimentum sollicitudin. Vestibulum sit amet interdum tellus. Quisque vitae varius nibh, id lobortis quam.
```
Comment (vchrombie) [inline Community/alumni.md:45]: You can add the project description from here, https://developers.google.com/season-of-docs/docs/2020/participants/project-linuxfoundation-jaskiratsingh2000

You don't need the whole thing. How about just the initial paragraphs?

File: Community/alumni.md [hunk 6]
```diff
@@ -2,33 +2,61 @@
...
+|[Nishchith K Shetty](#nishchith-k-shetty)|2019|GSoC|Support of Source Code Related Metrics|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
+|[Keanu Nichols](#keanu-nichols)|2018|GSoC|Reporting of CHAOSS Metrics|Sean Goggins, Jesus Gonzalez-Barahona|
+|[Pranjal Aswani](#pranjal-aswani)|2018|GSoC|Reporting of CHAOSS Metrics: Refactoring the existing code and extending the capabilities of the Manuscripts Project|Valerio Cosentino, Jesus Gonzalez-Barahona|
 
 
 ## Mentees
 
+### Jaskirat Singh
+
+Jaskirat Singh selected for GSoD 2020
```
Comment (vchrombie) [inline Community/alumni.md:36]: How about mentioning the Linux Foundation here?
`Jaskirat Singh was selected for GSoD 2020 under the Linux Foundation.`

WDYT @klumb?

File: Community/alumni.md [hunk 7]
```diff
@@ -2,33 +2,61 @@
...
+Xiaoya Esther selected for GSoD 2020
+
+#### Project Title
+
+Build documentation for CHAOSS D&I Badging project
+
+
+#### Project Description
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam faucibus sapien nisl, sit amet semper urna imperdiet sit amet. Mauris tempor, dui eu placerat dapibus, sapien mi maximus purus, in porta nibh ante et turpis. Donec ac vestibulum enim. Donec ut iaculis lacus. Phasellus consectetur elementum vulputate. Donec pulvinar lectus sit amet condimentum sollicitudin. Vestibulum sit amet interdum tellus. Quisque vitae varius nibh, id lobortis quam.
```
Comment (vchrombie) [inline Community/alumni.md:59]: You can add the project description from here, https://developers.google.com/season-of-docs/docs/2020/participants/project-linuxfoundation-xiaoya

Again, you don't need the whole thing. You can add the paras which give a good sense of the project

**Review by vchrombie — COMMENTED** [review]
Thanks for the PR @ritik-malik 

I have some comments with some minor changes, and also few suggestions. I would leave the PR to @klumb for his replies and review.

**Comment by ritik-malik** [comment]
@vchrombie @klumb
I pushed the changes as suggested in the review

**Comment by klumb** [comment]
@vchrombie is this ready to merge?

**Comment by vchrombie** [comment]
Hi @klumb, it looks good to me. Thanks for asking, you can check the PR. :smile:

**Pull request merged**